src/commands/Moderation/cleanban.ts

```diff
@@ -52,7 +52,7 @@ export default class extends Command {
 		if (bannedUser) {
 			await context.channel.send(
 				await this.client.bulbutils.translate("already_banned", context.guild?.id, {
-					target,
+					target: target.user,
```

Comment:
This branch probably fixes some bugs just by existing. For example, the wrong type was being passed here which means this line probably was printing "undefined" in some user-facing messages